src/styles/blocks/card.scss

```diff
@@ -0,0 +1,72 @@
+.card {
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding-block: 32px 16px;
+  padding-inline: 16px;
+
+  &__image {
+    display: flex;
+    width: 100%;
+    justify-content: center;
+
+    &-img {
+      width: 160px;
+      height: 134px;
+      background-image: url(/src/images/imac.jpeg);
+      background-size: 100% 100%;
+    }
+  }
+
+  &__description {
+    margin-block: 40px 0;
```

Comment:
The `margin-block` property is not widely used and might not be supported in all browsers. It is recommended to use `margin-top` and `margin-bottom` for better cross-browser compatibility.